src/Illuminate/Validation/Concerns/FormatsMessages.php

```diff
@@ -3,6 +3,7 @@
 namespace Illuminate\Validation\Concerns;
 
 use Closure;
+use Illuminate\Http\File;
```

Comment:
As suggested below, I'd suggest to use Symfonys file class to check against, as thats the one used inside the ValidatesAttributes class.

```suggestion
use Symfony\Component\HttpFoundation\File\File;
```